test/unit/clients/connector_client_charge_tests.js

```diff
@@ -30,30 +31,55 @@ describe('connector client - charge tests', function () {
   before(() => provider.setup())
   after(() => provider.finalize())
 
+  describe('making auth request', function () {
+    const authRequest = fixtures.validAuthorisationRequest()
+    
+    before(() => {
+      const response = fixtures.validChargeCardDetailsAuthorised()
+      const builder = new PactInteractionBuilder(AUTHORISE_CHARGE_URL)
+        .withRequestBody(authRequest.getPactified())
+        .withMethod('POST')
+        .withState('a sandbox account exists with a charge with id testChargeId that is in state ENTERING_CARD_DETAILS.')
```

Comment:
The `worldpay_3ds_flex_ddc_result` is only applicable to worldpay accounts, should that be reflected in the state rather than using `sandbox`?

Reply:
I did think about that but thought it wouldn't work to have worldpay, otherwise the provider test would actually try to call worldpay...